Document that APM and Logs are optional when using IAST (#36221)
DataDog/documentation: content/en/security/code_security/iast/setup/_index.md

1. **Datadog Agent Installation:** The Datadog Agent is installed and configured for your application's operating system or container, cloud, or virtual environment.
2. **Supported Tracing Library:** The Datadog Tracing Library used by your application or service supports Runtime Code Analysis (IAST) capabilities for the language of your application or service. For more details, see the **Compatibility Requirements** section below.

> **Note**: The Datadog Agent and tracing library are the only requirements for IAST. APM tracing and Log Management are not required — IAST findings are reported independently. You can optionally keep APM and Logs enabled alongside IAST, or disable them explicitly. See [Disable APM and Logs](#disable-apm-and-logs) for instructions.

Reviewer: did you intentionally use "Log Management" instead of "Infrastructure Monitoring"?

The last sentence then links to a different page to see the disablement instructions but they are actually inlined in this page below too - is it intentional?

Author: 1. "Log Management" was a mistake — updated to "Infrastructure Monitoring" throughout.
2. The link — you're right, the instructions are inlined on this page in the "Disable APM and Infrastructure Monitoring" section. The link in the Prerequisites note now points to the #disable-apm-and-infrastructure-monitoring anchor on this same page, so it scrolls down rather than navigating away.

> **Note**: The Datadog Agent and tracing library are the only requirements for IAST. APM tracing and Infrastructure Monitoring are not required — IAST findings are reported independently. You can optionally keep APM and Infrastructure Monitoring enabled alongside IAST, or disable them explicitly. See [Disable APM and Infrastructure Monitoring](#disable-apm-and-infrastructure-monitoring) for instructions.

Reviewer 2: ```suggestion
<div class="alert alert-info">The Datadog Agent and tracing library are the only requirements for IAST. APM tracing and Infrastructure Monitoring are not required; IAST reports findings independently of both. To disable APM or Infrastructure Monitoring, see <a href="#disable-apm-and-infrastructure-monitoring">Disable APM and Infrastructure Monitoring</a>.</div>
```

[4]: https://app.datadoghq.com/security/code-security/iast
[5]: /help

## Disable APM and Infrastructure Monitoring

> IAST requires the Datadog Agent and tracing library, but does not require APM tracing or Infrastructure Monitoring. If you want to run IAST without sending APM traces or infrastructure metrics to Datadog, you can disable them explicitly.

Reviewer 2: ```suggestion
To run IAST without APM traces or infrastructure metrics, disable them with the environment variables below.
```

### Disable APM tracing

To disable APM tracing on the tracer while keeping IAST enabled, set `DD_APM_TRACING_ENABLED=false`:

```shell
DD_IAST_ENABLED=true
DD_APM_TRACING_ENABLED=false
DD_SERVICE=<YOUR_SERVICE_NAME>
DD_ENV=<YOUR_ENVIRONMENT>

> ```

Reviewer 2: ```suggestion
{{< code-block lang="bash" >}}
DD_IAST_ENABLED=true
DD_APM_TRACING_ENABLED=false
DD_SERVICE=<YOUR_SERVICE_NAME>
DD_ENV=<YOUR_ENVIRONMENT>
{{< /code-block >}}
```

### Disable infrastructure monitoring

To disable infrastructure metrics collection on the Datadog Agent, set `DD_INFRASTRUCTURE_MODE=none` in your Agent configuration:

```shell
DD_INFRASTRUCTURE_MODE=none

> ```

Reviewer 2: ```suggestion
{{< code-block lang="bash" >}}
DD_INFRASTRUCTURE_MODE=none
{{< /code-block >}}
```
